src/main/resources/application.yaml

```diff
@@ -0,0 +1,12 @@
+spring:
+  jpa:
+    database: postgresql
+    show-sql: true
+    hibernate:
+      ddl-auto: create
```

Comment:
Since we are managing database schema via migration tool I don't think it makes sense to set this config to `create`.
You can find detailed docs [here](https://docs.jboss.org/hibernate/orm/5.2/userguide/html_single/Hibernate_User_Guide.html#configurations-hbmddl)

Reply:
So it would be better to use an update here or none?

Reply:
I think to don't need this check at all so `none` would be the best choice